concent_api/verifier/decorators.py

```diff
@@ -24,3 +28,38 @@ def wrapper(*args, **kwargs):
             # Remove any files left in VERIFIER_STORAGE_PATH.
             clean_directory(settings.VERIFIER_STORAGE_PATH)
     return wrapper
+
+
+def ensure_retry_of_locked_calls(func):
+    @wraps(func)
+    def wrapper(*args, **kwargs):
+        repeat_handler = RepeatHandler()
+        wait_time_in_sec = 0.05
+        should_repeat = True
+        result = None
+
+        while should_repeat:
+            try:
+                result = func(*args, **kwargs)
+                should_repeat = False
+            except DatabaseError:
+                repeat_handler.is_max_number_of_tries_exceeded()
+                time.sleep(wait_time_in_sec)
+        return result
+    return wrapper
+
+
+class RepeatHandler:
+
+    def __init__(self):
+        self.max_number_of_retries = 100
+        self.current_number_of_retries = 0
+
+    def is_max_number_of_tries_exceeded(self):
```

Comment:
This function starts with `is_...` which suggests that it returns a boolean value but it does not return anything.